### PR TITLE
[api][backend] add package releasename information

### PR DESCRIPTION
**Pull request by adrianschroeter**
needed for transporting the origin package container name.
This is a preparation for fixing the channel collection for
live patch updates

### CODE REVIEW

**Comment by adrianschroeter** [comment]
Just for review purposes here atm. Needs another fix in the backend.

(This is one out of too urgent needed bugfixes for maintenance team)

File: src/api/app/models/bs_request_action.rb [hunk 1]
```diff
@@ -499,6 +499,8 @@ def check_maintenance_release(pkg, repo, arch)
   end
 
   def create_expand_package(packages, opts = {})
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
```
Comment (hennevogel) [inline src/api/app/models/bs_request_action.rb]: You never enable them again. That means they are off for the rest of the file.

**Comment by adrianschroeter** [comment]
it includes now the other fix. I want to extend the test case for kgraft diffing a bit though. But can also happen later.

**Comment by adrianschroeter** [comment]
still important, but too risky for tomorrows deployment.

**Comment by adrianschroeter** [comment]
k, deployed, we should take it for next deployment in two weeks.

**Pull request merged**